browser-test/src/trusted_intermediary.test.ts

```diff
@@ -1033,4 +1033,22 @@ test.describe('Trusted intermediaries', () => {
       )
     })
   })
+  test.describe('organization members table', () => {
+    test('shows name, email and account status', async () => {
+      const {page, tiDashboard} = ctx
+      await loginAsTrustedIntermediary(page)
+      await tiDashboard.gotoTIDashboardPage(page)
+      await waitForPageJsLoad(page)
+
+      await validateScreenshot(
+        page.getByTestId('org-members-table'),
+        'org-members-table',
+      )
```

Comment:
praise: I like just testing a particular component of a page -- it means we probably need to make fewer screenshot updates in other PRs since this screenshot is so focused.